project/AkkaBuild.scala

```diff
@@ -132,28 +132,72 @@ object AkkaBuild {
     /**
      * Test settings
      */
+    fork in Test := true,
+
+    // default JVM config for tests
+    javaOptions in Test ++= {
+      val defaults = Seq(
+        // ## core memory settings
+        "-XX:+UseG1GC",
+        // most tests actually don't really use _that_ much memory (>1g usually)
+        // twice used (and then some) keeps G1GC happy - very few or to no full gcs
+        "-Xms3g", "-Xmx3g",
+        // increase stack size (todo why?)
+        "-Xss2m",
+
+        // ## extra memory/gc tuning
+        // this breaks jstat, but could avoid costly syncs to disc see http://www.evanjones.ca/jvm-mmap-pause.html
+        "-XX:+PerfDisableSharedMem",
+        // tell G1GC that we would be really happy if all GC pauses could be kept below this as higher would
+        // likely start causing test failures in timing tests
+        "-XX:MaxGCPauseMillis=300",
+        // nio direct memory limit (todo why this value?)
+        "-XX:MaxDirectMemorySize=256m",
```

Comment:
the reason for increasing that might be that it's needed for some Artery tests (Aeron)